scalameta/contrib/src/main/scala/scala/meta/contrib/implicits/implicits.scala

```diff
@@ -0,0 +1,3 @@
+package scala.meta.contrib.implicits
+
+trait implicits extends Statements.ToStatementOps
```

Comment:
I think we should opt for a similar strategy as scalameta and aim for a single a la carte import scala.meta.contrib._